shared/client/src/client.rs

```diff
@@ -266,22 +267,28 @@ impl<T: NodeIdentity, A: AuthenticatableIdentity + 'static, B: Backend<T> + 'sta
                                         let _ = trace_span!("NetworkEvent::DownloadComplete", hash = %hash).entered();
                                         metrics.record_download_completed(hash, from);
                                         if retried_downloads.remove(hash).await.is_some() {
-                                            debug!("Successfully downloaded previously failed blob {}", hex::encode(hash));
+                                            info!("Successfully downloaded previously failed blob {}", hex::encode(hash));
                                         }
                                         match download_data {
                                             TransmittableDownload::DistroResult(distro_result) => {
                                                 debug!("Download complete: step {} batch id {}", distro_result.step, distro_result.batch_id);
                                                 run.apply_distro_result(hash, distro_result, None);
                                             },
                                             TransmittableDownload::ModelParameter(parameter) => {
-                                                debug!("Download complete: parameter {}", parameter.name()?);
+                                                current_downloaded_parameters += 1;
+                                                info!("Download complete: parameter {}", parameter.name()?);
+                                                if let Some(total_parameters) = total_parameters {
+                                                    info!("Downloaded parameters total: {}/{}", current_downloaded_parameters, total_parameters);
+                                                } else {
+                                                    error!("Total parameters not set");
+                                                }
```

Comment:
I will add a little comment explaining that this should never really happen, it just a safeguard log in case something really weird happens

Reply:
It’s just a variable we set at the beginning of the param request process to help with logging each completed download. It shouldn’t happen, but if the variable isn’t set correctly while the actual list of parameters is still fine, then the process can continue, we just won’t be able to log the total number of parameters.